docs/releases/2.0.txt

```diff
@@ -104,7 +104,7 @@ Minor features
 :mod:`django.contrib.postgres`
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-* The new ``distinct`` argument for
+* The new ``distinct`` parameter for
```

Comment:
Refers to https://github.com/django/django/blob/487362fa8f23d41de4db58e7408e66eb36399af0/django/contrib/postgres/aggregates/general.py#L13